pkg/systemlogmonitor/log_monitor.go

```diff
@@ -165,7 +193,15 @@ func (l *logMonitor) generateStatus(logs []*logtypes.Log, rule systemlogtypes.Ru
 				break
 			}
 		}
+		if *l.config.EnableMetricsReporting {
+			err := problemmetrics.GlobalProblemMetricsManager.NewPermanentProblem(rule.Condition, rule.Reason, true)
```

Comment:
SG. Done.

In that case, I will rename `NewTemporaryProblems()` and `NewPermanentProblem()` to `IncrementProblemCounter()` and `SetProblemGauge()`.
Because new we are now reporting to problem counter for permanent problems as well.